lib/model-client.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const Model = require('@janiscommerce/model');
+
+class Client extends Model {
+
+	get databaseKey() {
+		return 'newClients';
+	}
+
+	static get table() {
+		return 'clients';
+	}
+
+	static get uniqueIndexes() {
+		return [
+			'id',
+			'code'
+		];
+	}
+
```

Comment:
Sumar método `shouldCreateLogs()` como **false**... no queremos que este modelo deje logs en nuestros servicios, y sino se pisará.
Para más info ver: https://www.npmjs.com/package/@janiscommerce/model#you-can-disable-this-functionality-by-setting-the-static-getter-shouldcreatelogs-to-false